### ISSUE TITLE
Low expires for JWT token. Why? Configurable?

### ISSUE THREAD

**Issue by WilliamDEdwards** [sensu/sensu-go#5072]
Since upgrading to Sensu 6.12, scripts running `sensuctl` are failing with:

     Error: API error: failed to request new refresh token; client returned 'The server returned the error: 401 Request unauthorized'

Looking at `.config/sensu/sensuctl/cluster` (where `sensuctl` stores tokens), I discovered that `expires_at` is set 5 minutes. 

First of all: was the validity period changed? I can't tell, as this version has no documentation nor release notes: https://github.com/sensu/sensu-docs/issues/4215. We've not experienced any issues before Sensu 6.12, though.

Second of all: is the validity period changeable? It's overly short, and therefore impractical. We don't call `sensuctl` every 5 minutes, nor do we want to.  https://docs.sensu.io/sensu-go/latest/observability-pipeline/observe-schedule/backend/#backend-configuration-options doesn't show a configuration option.

**Comment by jhenderson-pro** [comment]
Hey, William, 

We've added a request to have this updated in the next release to a configurable amount of time.

**Comment by StefanTT** [comment]
The very annoying problem with this 5min timeout is that the web UI also uses the token and the token refresh timer does not reliably work. Sometimes it runs when the window is left alone, sometimes it does not.

I would create an issue for the web repository but looking at the last change this seems to abandoned.

**Comment by eike-spiller** [comment]
adding "--access-token-expiry 1440m --refresh-token-expiry 10080m" to" sensu-backend start" seems to work (1day/1week) as of 6.13.0, thanks!

**Comment by ipaqmaster** [comment]
Also having this issue. Sensu-go seems abandoned.

**Comment by WilliamDEdwards** [comment]
> Sensu-go seems abandoned.

Yup, Sensu turned into abandonware. Shame.

**Comment by jhenderson-pro** [comment]
Hey folks, we're here!  We've fixed this issue in a previous release: [6.13.0](https://docs.sensu.io/sensu-go/latest/release-notes/#6130-release-notes) on April 10th:

```
NEW FEATURES:

    Added access-token-expiry (in minutes) backend configuration variable to control expiry of access token.
    Added refresh-token-expiry (in minutes) backend configuration variable to control expiry of refresh token.
```

**Comment by WilliamDEdwards** [comment]
> Hey folks, we're here! We've fixed this issue in a previous release: [6.13.0](https://docs.sensu.io/sensu-go/latest/release-notes/#6130-release-notes) on April 10th:
> 
> ```
> NEW FEATURES:
> 
>     Added access-token-expiry (in minutes) backend configuration variable to control expiry of access token.
>     Added refresh-token-expiry (in minutes) backend configuration variable to control expiry of refresh token.
> ```

@jhenderson-pro 

This doesn't actually work. Setting `access-token-expiry` and `refresh-token-expiry` in `/etc/sensu/backend.yml` to 2880 (48 hours in minutes) causes the token to expire at the exact same time it's requested, as does 21600 (6 hours in seconds; thought I'd try a lower value, with a different unit). 

    sensu-backend version 6.13.1+ee, enterprise edition, build d3c9eeb0f68d3b82ebed60116a157dd13ee9c3d7, built 2025-05-28T10:47:57Z, built with go1.23.4

**Comment by kristiangronas** [comment]
I suggest explicitly using the units of time.ParseDuration https://go-language.org/go-docs/time/#ParseDuration

viper's default is in nanoseconds

**Comment by WilliamDEdwards** [comment]
> I suggest explicitly using the units of time.ParseDuration https://go-language.org/go-docs/time/#ParseDuration
> 
> viper's default is in nanoseconds

You're right, suffixing with 'm' works. So the documentation is wrong ("in minutes").
